### DISCUSSION TITLE
Github pages keeps re-enabling

### DISCUSSION THREAD

**Discussion by FormulaRossa** [community/community#50497] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello,

I unpublished my GitHub pages site, but it keeps re-enabling on every push, and it's still visible under deployments.

Does somebody know what I can do?

Repo with this problem: https://github.com/FormulaRossa/wiki

**Comment by yoannchaudet** [comment]
You are probably just using the "Unpublish" option which does not disable Pages but just remove the current deployment. We have some work planned to streamline and make this more explicit.

To disable Pages you want to select the "None" branch. Currently it is still configured to track changes on your `master` branch (`/docs` folder).

**Reply by FormulaRossa** [reply]
Thank you!
That was the problem.
